source/_integrations/flipr.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Flipr
+description: Instructions on how to integrate your Flipr device within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2021.2
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@cnico'
+ha_domain: flipr
+---
+
+[Flipr](https://www.goflipr.com) is a smart pool monitor that publishes data to the cloud via SigFox.
+Flipr sends data like ph, chlorine or temperature to a cloud server on a regular basis in order to monitor your pool.
+This integration gives you access to these information on Home Assistant in addition to the vendor's smartphone application.
+
+There is currently support for the following information within Home Assistant:
+
+- Chlorine Level
+- pH
+- Water Temperature
+- Red OX Level
+- Last measure date
+- pH Status Indicator
+- Chlorine Status Indicator
+
+## Configuration
+
+You will need to use the standalone app for this device to register a username and password.
+
+- [Google](https://play.google.com/store/apps/details?id=com.goflipr.flipr)
+- [Apple](https://apps.apple.com/fr/app/flipr/id1225898851)
+
+Home Assistant offers Flipr integration through **Configuration** -> **Integrations**. Click the `+` sign and from the list of integrations select **Flipr**. Simply enter `username` and `password` when prompted. Your flipr ID is automatically determined.
+
+## Tips
```

Comment:
Let's remove this part/chapter, as it is possibly confusing.
For example, entity IDs are not guaranteed.
And in general, we don't recommend using badges.

Reply:
OK, I commited your proposal.